docs/flowcollector-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -159,11 +159,11 @@ Type::
 
 | `cacheActiveTimeout`
 | `string`
-| cacheActiveTimeout is the max period during which the reporter will aggregate flows before sending
+| cacheActiveTimeout is the max period during which the reporter will aggregate flows before sending Increasing `cacheMaxFlows` and `cacheActiveTimeout` can decrease the network traffic overhead and the CPU load, however you can expect higher memory consumption and an increased latency in the flow collection.
```

Comment:
```suggestion
| cacheActiveTimeout is the max period during which the reporter will aggregate flows before sending. Increasing `cacheMaxFlows` and `cacheActiveTimeout` can decrease the network traffic overhead and the CPU load, however you can expect higher memory consumption and an increased latency in the flow collection.
```

Reply:
Just added a period

Reply:
thx! I need to amend the PR as there are other files to update